cmp/7e7d33d5-7101-47f6-a358-1126c7876aa4/component.lp

```diff
@@ -0,0 +1,94 @@
+(librepcb_component 7e7d33d5-7101-47f6-a358-1126c7876aa4
+ (name "Buffer With Inverted Enable Quad")
+ (description "Quad buffer/line driver; 3-state")
+ (keywords "74HCT125, 74HC125, 74XX125, buffer, driver, line-driver, quad-buffer, quad buffer")
+ (author "Shibin Das")
+ (version "0.1")
+ (created 2019-04-07T20:07:35Z)
+ (deprecated false)
+ (category 56f01b1d-0723-4c33-88f7-9968cf96b50f)
+ (category a4c3b94b-5321-4adb-8a77-d3b53f178cee)
+ (schematic_only false)
+ (default_value "74XX125")
```

Comment:
A placeholder should be used to dynamically take the name from the device as value. Unfortunately this is not clearly documented yet, but LibrePCB 0.1.1 now helps you to choose the proper value if this field is empty :)

```suggestion
 (default_value "{{PARTNUMBER or DEVICE}}")
```